QUDAConfig.cmake.in

```diff
@@ -6,14 +6,31 @@ set(QUDA_QMP @QUDA_QMP@)
 set(QUDA_MPI @QUDA_MPI@)
 set(QUDA_QIO @QUDA_QIO@)
 set(QUDA_OPENMP @QUDA_OPENMP@)
+set(QUDA_QDPJIT @QUDA_QDPJIT@)
+
+set(QUDA_TARGET_CUDA @QUDA_TARGET_CUDA@)
+set(QUDA_TARGET_HIP  @QUDA_TARGET_HIP@)
+
+set(QUDA_NVSHMEM  @QUDA_NVSHMEM@)
```

Comment:
Wondering why this is here? nvshmem is consumed by QUDA so no calling app should care about this.